easybuild/easyconfigs/g/GTOOL/GTOOL-0.7.5.eb

```diff
@@ -0,0 +1,24 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Tarball"
+
+name = 'GTOOL'
+version = '0.7.5'
+
+homepage = 'http://www.well.ox.ac.uk/~cfreeman/software/gwas/gtool.html'
+description = """ GTOOL is a program for transforming sets of genotype data for use with 
+the programs SNPTEST and IMPUTE.  """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://www.well.ox.ac.uk/~cfreeman/software/gwas/']
+sources = ['gtool_v%(version)s_x86_64.tgz']
+
+sanity_check_paths = {
+    'files': ["gtool", "LICENCE"],
```

Comment:
@rtripath89 what about making sure the install directory is added to `$PATH`, so `gtool` is easily available after loading the generated module?

You'll need to include this:

```python
modextrapaths = {'PATH': ''}
```

(note: this was done automagically with the `Binary` easyblock, but not with `Tarball` unless the binaries are in